tasks/Module 2/task_1.py

```diff
@@ -0,0 +1,56 @@
+"""Module 2 tasks."""
+
+from collections import defaultdict
+
+ITEMS_TO_SKIP = ['chewed gum', 'rubbish', 'used tissue']
+MAX_INVENTORY_CAP = 80
+HEAVY_WEIGHT_THRESHOLD = 70
+LIGHT_WEIGHT_THRESHOLD = 60
+
+
+def display_inventory(inventory):
+    """Display the contents of the inventory."""
+    item_total = 0
+    print("Inventory:")
+    for item_name, item_count in inventory.items():
+        print(item_count, item_name)
+        item_total += item_count
+    print(f"Total number of items: {item_total}")
+    if item_total >= MAX_INVENTORY_CAP:
+        print("CAUTION: You are overloaded, can't move!")
+    elif item_total >= HEAVY_WEIGHT_THRESHOLD:
+        print("CAUTION: Your equipment is very heavy, you're moving slower than usual!")
+    elif item_total >= LIGHT_WEIGHT_THRESHOLD:
+        print("CAUTION: Your backpack weighs a lot, your stamina runs out quicker!")
+
+
+def add_to_inventory(inventory, added_items, items_to_skip):
+    """
+    Ad useful items to the inventory and skip not useful.
```

Comment:
We have a typo „Ad” -> Add

Also we classify our items as useful/not useful which is subjective and overly descriptive. This function does not classify items as useful/not useful, it simply skip items provided by user as to-be-skipped. 

Naming could also change from added_items to items, as those items are about to be added to be precise.

To sum up, this function adds all items except for those marked explicitly as to be skipped.